src/main/kotlin/org/hyacinthbots/lilybot/database/migrations/main/mainV5.kt

```diff
@@ -0,0 +1,15 @@
+package org.hyacinthbots.lilybot.database.migrations.main
+
+import org.hyacinthbots.lilybot.database.entities.ThreadData
+import org.litote.kmongo.coroutine.CoroutineDatabase
+import org.litote.kmongo.exists
+import org.litote.kmongo.setValue
+
+suspend fun mainV5(db: CoroutineDatabase) {
+	db.createCollection("autoThreadingData")
+
+	// todo actually set values for guildId
```

Comment:
Oh dear lord help us. This requires some funky magic in the startup hooks file for the guild id to be actually gotten.